lib/test/t_ComposedDistribution_std.cxx

```diff
@@ -75,7 +75,10 @@ int main(int, char *[])
     distribution.setName("myDist");
     UnsignedInteger dim = distribution.getDimension();
     fullprint << "Distribution " << distribution << std::endl;
```

Comment:
This is untrue.
The code:
```cpp
    fullprint << "Distribution " << distribution << std::endl;
```
triggers the `__repr__()` C++ code, which produces:
```
Distribution class=ComposedDistribution name=myDist dimension=3 copula=class=IndependentCopula name=Independent copula dimension=3 marginal[0]=class=Normal name=First dimension=1 mean=class=Point name=Unnamed dimension=1 values=[1] sigma=class=Point name=Unnamed dimension=1 values=[2] correlationMatrix=class=CorrelationMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[1] marginal[1]=class=Normal name=Second dimension=1 mean=class=Point name=Unnamed dimension=1 values=[2] sigma=class=Point name=Unnamed dimension=1 values=[3] correlationMatrix=class=CorrelationMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[1] marginal[2]=class=Normal name=Third dimension=1 mean=class=Point name=Unnamed dimension=1 values=[3] sigma=class=Point name=Unnamed dimension=1 values=[4] correlationMatrix=class=CorrelationMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[1]
```
The code:
```cpp
    std::cout << "Distribution " << std::endl;
    std::cout << distribution << std::endl;
```
triggers the `__str__()` C++ code, which produces:
```
Distribution 
ComposedDistribution(Normal(mu = 1, sigma = 2), Normal(mu = 2, sigma = 3), Normal(mu = 3, sigma = 4), IndependentCopula(dimension = 3))
```
Since the two codes are different, the two unit tests are meaningful. Please tell me if I am right, so that I can resolve that discussion.